BREEZE.md

```diff
@@ -0,0 +1,356 @@
+<!--
```

Comment:
Most of our documentation is in RST format. Should we stick to one format?  I prefer RST, because then the built documentation can be versioned and available to users.
Reference: https://github.com/apache/airflow/pull/4827

Reply:
Sure. I will change it @mik-laj .

Reply:
Moved to https://github.com/apache/airflow/pull/4932. I will change it there.

Reply:
Fixed in #4932 @mik-laj